Plugins/Flow.Launcher.Plugin.Program/Programs/UWP.cs

```diff
@@ -601,92 +601,114 @@ internal string LogoPathFromUri(string uri)
-                        {
-                            PackageVersion.Windows10, new List<int>
-                            {
-                                100,
-                                125,
-                                150,
-                                200,
-                                400
-                            }
-                        },
-                        {
-                            PackageVersion.Windows81, new List<int>
-                            {
-                                100,
-                                120,
-                                140,
-                                160,
-                                180
-                            }
-                        },
+                        if (File.Exists(path))
                         {
-                            PackageVersion.Windows8, new List<int>
-                            {
-                                100
-                            }
+                            return path; // shortcut, avoid enumerating files
                         }
-                    };
 
-                    if (scaleFactors.ContainsKey(Package.Version))
-                    {
-                        foreach (var factor in scaleFactors[Package.Version])
+                        var logoNamePrefix = Path.GetFileNameWithoutExtension(uri); // e.g Square44x44
+                        var logoDir = Path.GetDirectoryName(path);  // e.g ..\..\Assets
+                        if (String.IsNullOrEmpty(logoNamePrefix) || String.IsNullOrEmpty(logoDir) || !Directory.Exists(logoDir))
                         {
-                            paths.Add($"{prefix}.scale-{factor}{extension}");
+                            // Known issue: Edge always triggers it since logo is not at uri
+                            ProgramLogger.LogException($"|UWP|LogoPathFromUri|{Package.Location}" +
+                               $"|{UserModelId} can't find logo uri for {uri} in package location (logo name or directory not found): {Package.Location}", new FileNotFoundException());
+                            return string.Empty;
                         }
-                    }
 
-                    var selected = paths.FirstOrDefault(File.Exists);
-                    if (!string.IsNullOrEmpty(selected))
-                    {
-                        return selected;
+                        var files = Directory.EnumerateFiles(logoDir);
+
+                        // Currently we don't care which one to choose
+                        // Just ignore all qualifiers
+                        // select like logo.[xxx_yyy].png
+                        // https://learn.microsoft.com/en-us/windows/uwp/app-resources/tailor-resources-lang-scale-contrast
+                        var selected = files.FirstOrDefault(file =>
```

Comment:
maybe we should prefer $44\times44$ if possible to reduce image size? Though it's not a big deal if it is hard.

Reply:
Try to find closest to 44x44 in the new commit. It seems to work well.